scripts/mixins/families/avatar.lua

```diff
@@ -27,7 +27,13 @@ g_mixins.families.avatar = function(avatarMob)
 
         -- When GM is enabled, mobs will not automatically engage.  Update Enmity one more
         -- time to ensure that the listener will actually be triggered.
-        mob:updateEnmity()
+        local master = mob:getMaster()
```

Comment:
Just wanted to make sure this was tested here.  The avatar may not be the mob's "pet", especially in cases where an elemental is also valid.

Reply:
> Just wanted to make sure this was tested here. The avatar may not be the mob's "pet", especially in cases where an elemental is also valid.

Can you think of any spots in game I can test this? I tested in Temenos and Jungle Boogeyman since that seems to be where this originated from. Both were able to hit me with GM toggled on.

Reply:
If Jungle worked and still had the avatar AF with elemental up, its a 👍 in my book

Reply:
![image](https://github.com/LandSandBoat/server/assets/166072302/0deefa41-c08f-4c44-8396-c1d1d564d540)

Yeah should be good.